PetFeeding/src/main/java/com/epam/dao/Dao.java

```diff
@@ -0,0 +1,15 @@
+package com.epam.dao;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface Dao<T, Id> {
+
+    Optional<T> getById(Id Id);
+
+    List<T> getAll();
+
+    T save(T t);
+
+    void delete(T t);
+}
```

Comment:
Please add newlines to the end of all files. Some tools require it

Reply:
fixed